search-app/src/cmr/search/site/data.clj

```diff
@@ -193,8 +204,18 @@
 
 (defmethod base-page :default
   [context]
-  (assoc (common-data/base-page context) :app-title "CMR Search"
-         :release-version (str "v " (common-config/release-version))))
+  (let [base-page (common-data/base-page context)
```

Comment:
Add comment on why we are doing the url manipulation. Without the background info, I don't know if you are doing it correctly.

Reply:
Done. Moved the comment to app-url->stac-urls function. Actually I forgot to remove the conversion code here after moving it to app-url->stac-urls. so it's redundant.